lib/Data/Mapper/ContentUpdateMapper.php

```diff
@@ -17,19 +18,23 @@ class ContentUpdateMapper implements FormDataMapperInterface
     /**
      * Maps a ValueObject from eZ content repository to a data usable as underlying form data (e.g. create/update struct).
      *
-     * @param ValueObject|\eZ\Publish\API\Repository\Values\Content\Content $contentDraft
+     * @param \eZ\Publish\API\Repository\Values\ValueObject|\eZ\Publish\API\Repository\Values\Content\Content $contentDraft
      * @param array $params
      *
-     * @return ContentUpdateData
+     * @return \EzSystems\RepositoryForms\Data\Content\ContentUpdateData
      */
     public function mapToFormData(ValueObject $contentDraft, array $params = [])
     {
         $optionsResolver = new OptionsResolver();
         $this->configureOptions($optionsResolver);
+
```

Comment:
Many empty lines ;)

Reply:
Done on purpose to separate logical blocks.

Reply:
As you know, I like them even too much :)